### PR TITLE
coreos-postinst: add CVE workaround for vmtoolsd

### PR DESCRIPTION
**Pull request by euank**

### CODE REVIEW

File: coreos-postinst [hunk 1]
```diff
@@ -127,6 +127,19 @@ if [ -e ${PLATFORM_PATH} ]; then
     fi
 fi
 
+# Our VMware OEM partition contained a version of vmtoolsd that was vulnerable to 
+# CVE-2015-5191 (http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5191).
+# This CVE can be mitigated by setting PrivateTmp=true on the vmtoolsd service file.
+# coreos-base/oem-vmware-10.1.5 is the last vulnerable version of the oem ebuild.
+VMTOOLSD_DROPIN=/etc/systemd/system/vmtoolsd.service.d/90-tmpfiles-cve-2015-4191.conf
+if [ ! -f $VMTOOLSD_DROPIN ] && grep -q coreos.oem.id=vmware /proc/cmdline; then
```
Comment (bgilbert) [inline coreos-postinst]: Old versions of the VMware OEM don't set `coreos.oem.id`, even after upgrades.  (Tested with 735.0.0.)  I think the correct approach is to also check `/etc/oem-release`; @crawford?

Comment (euank) [inline coreos-postinst]: Updated to check oem-release

File: coreos-postinst [hunk 2]
```diff
@@ -127,6 +127,22 @@ if [ -e ${PLATFORM_PATH} ]; then
...
 
+# Our VMware OEM partition contained a version of vmtoolsd that was vulnerable to 
+# CVE-2015-5191 (http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5191).
+# This CVE can be mitigated by setting PrivateTmp=true on the vmtoolsd service file.
+# coreos-base/oem-vmware-10.1.5 is the last vulnerable version of the oem ebuild.
+# Note: we check in both /etc/oem-release and /usr/share/oem/oem-release
+# because pre-ignition Container Linux machines did not set the oem.id cmdline,
+# and also wrote oem-release only to /etc
+VMTOOLSD_DROPIN=/etc/systemd/system/vmtoolsd.service.d/90-tmpfiles-cve-2015-4191.conf
+if [ ! -f $VMTOOLSD_DROPIN ] && grep --quiet --no-messages "^ID=vmware$" /etc/oem-release /usr/share/oem/oem-release; then
```
Comment (bgilbert) [inline coreos-postinst]: I'd suggest `-e` rather than `-f`.

File: coreos-postinst [hunk 3]
```diff
@@ -127,6 +127,22 @@ if [ -e ${PLATFORM_PATH} ]; then
...
+    mkdir -p /etc/systemd/system/vmtoolsd.service.d/
+    cat >$VMTOOLSD_DROPIN <<EOF
+[Service]
```
Comment (bgilbert) [inline coreos-postinst]: Is it worth adding a comment explaining why this dropin exists?

**Pull request merged**